core/test/integration/synchronization/ethereum_synchronization.cpp

```diff
@@ -347,6 +348,9 @@ TEST_F(EthereumLikeWalletSynchronization, ReorgLastBlock) {
     auto walletName = "e847815f-488a-4301-b67c-378a5e9c8a61";
     {
         auto fakeHttp = std::make_shared<test::FakeHttpClient>();
+
+        backend = std::static_pointer_cast<DatabaseBackend>(DatabaseBackend::getSqlite3Backend());
```

Comment:
Note to reviewer: this has to be done this way to be iso with the rest of the tests (they don’t pass the `usePostgreSQL` argument to `newDefaultPool`, so they use SQLite3).

Reply:
Extra note to reviewers: maybe the `static_pointer_cast` is not needed?